openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1088,10 +1088,10 @@ A Client submits a Batch Credential Request to the Batch Credential Endpoint by
   * `proofs`: OPTIONAL. See (#credential-request).
 * `credential_response_encryption`: OPTIONAL. Object containing information for encrypting the Batch Credential Response. It contains the same parameters as defined in #{credential-request}. If this request element is not present, the corresponding Batch Credential Response returned is not encrypted.
 
-Below is a non-normative example of a Batch Credential Request requesting:
+Below is a non-normative example of a Batch Credential Request requesting four Credentials:
 
-* 2 Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys;
-* 2 Credentials for the same Credential Configuration but with different Credential Dataset.
+* Two Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format;
+* Two Credentials for the same Credential Configuration but with different Credential Dataset in [@VC_DATA] format.
```

Comment:
@paulbastian can you please re-review if this new text makes sense

Reply:
Makes sense